### PR TITLE
Fix Misspellings across Cumulus

### PR DESCRIPTION
**Pull request by elisoncrum**
## Summary

Addresses no open issue.

Fixes 70 major and minor spelling errors within 54 files either .md, .js, or .ts files.

## Changes
- fixed several minor misspellings in current and past .md files with misspellings
- fixed possibly major misspelling within `packages/integration-tests/api/executions.js`
  - Line 46 incorrectly references `@cumulus/integration-tests/exeuctions.getExecutionStatus` which does not exist. Changes `exeuctions` to `executions`
- fixed test output and documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1720,7 +1720,7 @@ new `update-granules-cmr-metadata-file-links` task.
   - Changed `distribution_api_gateway_stage` variable for `cumulus` module to `tea_api_gateway_stage`
   - Changed `api_gateway_stage` variable for `distribution` module to `tea_api_gateway_stage`
 - **CUMULUS-2224**
-  - Updated `/reconciliationReport`'s file reconciliation to include `"EXTENDED METADATA"` as a valid CMR relatedUrls Type.
+  - Updated `/reconciliationReport`'s file reconciliation to included `"EXTENDED METADATA"` as a valid CMR relatedUrls Type.
```
Comment (flamingbear) [inline CHANGELOG.md:1723]: Nope

Comment (elisoncrum) [reply CHANGELOG.md]: Fixed in `2eb376f`

**Comment by flamingbear** [comment]
Thanks for finding these, I have to pull into my branch so it can go through CI.

**Comment by flamingbear** [comment]
Also thanks for fixing that one incorrect one.

**Pull request merged**